depth_image_topic: /XTION/depth/image 
camera_info_topic: /XTION/depth/camera_info 
#source: /home/mwuethri_local/data/complete_projects/coordinate_particle_filter/humanoids.2014/evaluation/moving_boxes/sim/dataset

rendering_root_left: L_SHOULDER
rendering_root_right: R_SHOULDER

collision: true

max_kl_divergence: 2.0

use_gpu: true
evaluation_count: 40 #100 #the number of evaluations

# filtering parameters ----------------------------------------------------------------------
downsampling_factor: 8 #8
initial_sample_count: 1 #10000 this is the number of samples for the initialization
max_sample_count: 10000 #10000 this is for the gpu to allocate memory

# process model parameters ------------------------------------------------------------------
#( 0: LPAN)  ( 1: LTILT) ( 2: LTILT_TOP) (3:  UPAN)  (4:  UTILT) (5:  LTILT_FRONT) 
#( 6: L_SFE) ( 7: L_SAA) ( 8: L_HR)      (9:  L_EB)  (10: L_WR)  (11: L_WFE)   (12: L_WAA) 
#(13: L_FR)  (14: L_LF)  (15: L_LF2)     (16: L_FR2) (17: L_RF)  (18: L_RF2)   (19: L_MF)  (20: L_MF2)     
#(21: R_SFE) (22: R_SAA) (23: R_HR)      (24: R_EB)  (25: R_WR)  (26: R_WFE)   (27: R_WAA) 
#(28: R_FR)  (29: R_LF)  (30: R_LF2)     (31: R_FR2) (32: R_RF)  (33: R_RF2)   (34: R_MF)  (35: R_MF2) 


joint_sigmas: [ 
 #0.01, 0.01, 0.01, 0.01, 0.01, 0.01,     #head
 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, #head
 0.02, 0.02, 0.02, 0.05, 0.05, 0.05, 0.05,      #left arm
 0.05, 0.05, 0.05, 0.05, 0.05, 0.05, 0.05, 0.05, #left hand
 0.02, 0.02, 0.02, 0.05, 0.05, 0.05, 0.05,      #right arm
 0.05, 0.05, 0.05, 0.05, 0.05, 0.05, 0.05, 0.05, #right hand
 ]



sampling_blocks: [
[ 0,  1,  2,  3,  4, 5],          #head
[ 6,  7,  8,  9, 10, 11, 12],     #left arm
[13, 14, 15, 16, 17 ,18, 19, 20], #left hand
[21, 22, 23, 24, 25, 26, 27],     #right arm
[28, 29, 30, 31, 32, 33, 34, 35]  #right hand
]

#sampling_blocks: [
#[ 0,  1,  2,  3,  4,  5,         #head
  #6,  7,  8,  9, 10, 11, 12,     #left arm
 #13, 14, 15, 16, 17 ,18, 19, 20, #left hand
 #21, 22, 23, 24, 25, 26, 27,     #right arm
 #28, 29, 30, 31, 32, 33, 34, 35] #right hand
#]


#joint_sigmas: [ #left arm
#0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
#0.1, 0.1, 0.1, 0.1, 0.1, 0.1,
#0.1, 0.0, 0.0, 0.0, 0.0, 0.0,
#0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
#0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
#0.0, 0.0, 0.0, 0.0, 0.0, 0.0
#]

#joint_sigmas: [ #right arm
 #0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
 #0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
 #0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
 #0.0, 0.0, 0.0, 0.1, 0.1, 0.1,
 #0.1, 0.1, 0.1, 0.1, 0.0, 0.0,
 #0.0, 0.0, 0.0, 0.0, 0.0, 0.0
 #]

#joint_sigmas: [ #both arms
#0.01, 0.01, 0.01, 0.01, 0.01, 0.01,
#0.1, 0.1, 0.1, 0.1, 0.1, 0.1,
#0.1, 0.0, 0.0, 0.0, 0.0, 0.0,
#0.0, 0.0, 0.0, 0.1, 0.1, 0.1,
#0.1, 0.1, 0.1, 0.1, 0.0, 0.0,
#0.0, 0.0, 0.0, 0.0, 0.0, 0.0
#]

#sampling_blocks: [
 #[ 0], [ 1], [ 2], [ 3], [ 4], [ 5]
#,[ 6], [ 7], [ 8], [ 9], [10], [11]
#,[12], [13], [14], [15], [16], [17]
#,[18], [19], [20], [21], [22], [23]
#,[24], [25], [26], [27], [28], [29]
#,[30], [31], [32], [33], [34], [35]
#]

# sampling_blocks: [[
#   0,  1,  2,  3,  4,  5
# , 6,  7,  8,  9, 10, 11
# ,12, 13, 14, 15, 16, 17
# ,18, 19, 20, 21, 22, 23
# ,24, 25, 26, 27, 28, 29
# ,30, 31, 32, 33, 34, 35
# ]]






joint_angle_sigma: 0.01
damping: 0.0

# observation model parameters --------------------------------------------------------------
tail_weight: 0.01 #0.01, 0.998
model_sigma: 0.003 #0.003, 0.005
sigma_factor: 0.00142478 #0.0014247
 
# visibility process model stuff ------------------------------------------------------------
# initial prob of the source being the object for each pixels
initial_occlusion_prob: 0.1 #0.1
# prob of source being object given that one second ago the source was the object
p_occluded_visible: 0.1 #0.1
# prob of source being object given that one second ago source was not object
p_occluded_occluded: 0.7 #0.7



